src/main/app/case/case.ts

```diff
@@ -327,6 +327,7 @@ export interface Case {
   miamStart?: string;
   miamWillingness?: string;
   miamNotWillingExplnation?: string;
+  c1A_childAbductedBefore?: string;
```

Comment:
@monalisasarkar1 As discussed in the call today, we can get rid of this field from case.ts if there is no conditional mapping

FYI, @channa-patil

Reply:
There is conditional mapping for this field